wpiunits/src/main/java/edu/wpi/first/units/BaseUnits.java

```diff
@@ -16,6 +16,9 @@ private BaseUnits() {
   /** The standard unit of time, seconds. */
   public static final Time Time = new Time(null, 1, "Second", "s");
 
+  /** The standard unit of frequency, hertz. */
+  public static final Frequency Frequency = new Frequency(null, 1, "Hertz", "Hz");
```

Comment:
Just a warning, I think we need to be careful about adding `Frequency` as a base type and its relationship with `times()`, `divide()`, and the `inverse()` added in this PR.

I think I would expect that:
- `Measure<Frequency>.inverse() -> Measure<Time>`
- `Measure<U>.times(Measure<Frequency>) -> Measure<Velocity<U>>`
- `Measure<U>.divide(Measure<Frequency>) -> Measure<Mult<U, Time>>`?

Reply:
Yeah perhaps Frequency should not have its own type and should instead be represented by `Measure<Velocity<Dimensionless>>`. My only problem with that is it doesn't intuitively read as a frequency imo.